src/Zicht/Bundle/UrlBundle/Entity/UrlAlias.php

```diff
@@ -45,7 +45,7 @@ class UrlAlias
     protected $id;
 
     /**
-     * @ORM\Column(type="string")
+     * @ORM\Column(type="string", unique=true)
```

Comment:
Ik ben het eens met deze aanpassing, maar dit zal bij enkele klanten misgaan omdat de database op dat moment al dubbele entries bevat.

Kun je een script oid in de changelog zetten wat ge-copy-paste kan worden (bijvoorbeeld in een migration bestand) om het te fixen?  Anders gaat het geheid mis bij een deploy naar diab of artez

Reply:
staat bij docs

Reply:
:+1: